### PR TITLE
samples(cloud-sql): Update cloud sql sample apps to netcoreapp 3.1

### PR DESCRIPTION
**Pull request by jsimonweb**

### CODE REVIEW

File: cloud-sql/sql-server/.gcloudignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**/obj/
```
Comment (enocom) [inline cloud-sql/sql-server/.gcloudignore:1]: Should we drop this file from this PR?

File: cloud-sql/sql-server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022 Google LLC.
```
Comment (enocom) [inline cloud-sql/sql-server/Dockerfile:1]: Shall we put this in a separate PR?

File: cloud-sql/sql-server/.gcloudignore (hunk 1 above)
Comment (jsimonweb) [reply cloud-sql/sql-server/.gcloudignore]: Done.

File: cloud-sql/sql-server/Dockerfile (hunk 2 above)
Comment (jsimonweb) [reply cloud-sql/sql-server/Dockerfile]: Done.

**Review by enocom — APPROVED** [review]
I can't speak to the test failures, but the change otherwise seems good.

File: cloud-sql/mysql/CloudSql.csproj [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Google.Api.Gax" Version="3.7.0" />
     <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="4.4.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="MySql.Data" Version="8.0.28" />
+    <PackageReference Include="MySql.Data" Version="8.0.23" />
```
Comment (amanda-tarafa) [inline cloud-sql/mysql/CloudSql.csproj:10]: Any reason why you are bumping down from 8.0.28 to 8.0.23?

**Review by amanda-tarafa — APPROVED** [review]
LGTM with one tiny question.

**Comment by amanda-tarafa** [comment]
Failures are unrelated are already being tracked in:

- #1687 
- #1690 

@jsimonweb Let me know when this is ready for merging and I'll force merge.

**Comment by jsimonweb** [comment]
Ready for merging.

File: cloud-sql/postgres/CloudSql.csproj [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Google.Api.Gax" Version="3.7.0" />
     <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="4.4.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="MySql.Data" Version="8.0.28" />
```
Comment (amanda-tarafa) [inline cloud-sql/postgres/CloudSql.csproj:10]: But this one here does not make the deployment fail? Or maybe this one is not needed here as this is the PG sample?

Comment (jsimonweb) [reply cloud-sql/postgres/CloudSql.csproj]: >> maybe this one is not needed here as this is the PG sample?

Correct. This is not used. I've updated the CloudSql.csproj file for Postgres to remove this. I also removed the unnecessary package reference to Npgsql in the MySQL CloudSql.csproj file.

Comment (amanda-tarafa) [reply cloud-sql/postgres/CloudSql.csproj]: Perfect thanks!

**Review by amanda-tarafa — APPROVED** [review]
LGTM
